pytorch/image_classification/main.py

```diff
@@ -452,9 +452,9 @@ def _worker_init_fn(id):
     # elif args.data_backend == "dali-cpu":
     #     get_train_loader = get_dali_train_loader(dali_cpu=True)
     #     get_val_loader = get_dali_val_loader()
-    # elif args.data_backend == "syntetic":
-    #     get_val_loader = get_syntetic_loader
-    #     get_train_loader = get_syntetic_loader
+    elif args.data_backend == "syntetic":
```

Comment:
@dannyel2511, do we need this in the diff?

Reply:
This is intentional. I enabled the syntetic data loader, so we are able to test quickly on dev setup without the actual dataset, we would only have to add the `--data-backend syntetic` to the args.
I don't think it hurts to have it enabled, but I can revert it if needed.